### ISSUE TITLE
vm create parameter --ssh-cert doesn't resolve tildas

### ISSUE THREAD

**Issue by ahmetalpbalkan** [Azure/azure-xplat-cli#1679]
If I run command:

    azure vm create -e --ssh-cert=~/.ssh/id_rsa.pub [...]

I get error: 

    error:   ENOENT, no such file or directory '~/.ssh/id_rsa.pub'

This is a bug. `~` should have been resolved to `$HOME` on Linux/Mac.

**Comment by devigned** [comment]
Yes, it should.

**Comment by kmouss** [comment]
Investigating to always parse ~ to be /home/..

**Comment by ahmetalpbalkan** [comment]
I believe should be rather $HOME/%USERPROFILE%, not `/home/`. It's `/Users/[[username]]` on OS X.

**Comment by ruslangabitov** [comment]
There are few ways to set a command option in Azure CLI:
1) You can simply specify value right after option separated by space, like this: 
```
--ssh-cert ~/.ssh/id_rsa.pub
```
In this case CLI will resolve <code>~</code> to user's home directory.
![image](https://cloud.githubusercontent.com/assets/6327511/7714202/969c6682-fe31-11e4-951e-78159ddcc354.png)

2) Alternatively you can use '=', just like in the example above, but in this case CLI treats value as if it was a string. 
Basically <code>--ssh-cert=~/.ssh/id_rsa.pub</code> is the same thing as <code>--ssh-cert '~/.ssh/id_rsa.pub'</code>, so CLI will just pass string to a command without resolving <code>~</code>. 

This is not a 'vm create' command issue, it's just the way Azure CLI handles options.

**Comment by ahmetalpbalkan** [comment]
@ruslangabitov makes perfect sense. closing.
